.gitignore

```diff
@@ -1,7 +1,6 @@
 .idea/
 node_modules/
 .vscode/
-dist/
```

Comment:
Why do we need dist inside the repo?

Reply:
Its need temporary, i remove it before merge